### PR TITLE
Add io.github.kevinsmia1939.PySimpleCV

### PR DESCRIPTION
**Pull request by kevinsmia1939**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

This submission fix app name from com.github to io.github.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by kevinsmia1939** [comment]
bot, build io.github.kevinsmia1939.PySimpleCV

**Comment by kevinsmia1939** [comment]
bot, build io.github.kevinsmia1939.PySimpleCV

File: fix_metadata_release.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
++++ b/data/io.github.kevinsmia1939.PySimpleCV.metainfo.xml	2022-11-21 19:19:29.964657298 +0100
+@@ -32,8 +32,9 @@
+  <url type="bugtracker">https://github.com/kevinsmia1939/PySimpleCV/issues</url>
+  <content_rating type="oars-1.1"/>
+  <releases>
++     <release version="v0.5.3" date="2022-11-21"/>
+      <release version="v0.5.2" date="2022-11-21"/>
+      <release version="v0.5.1" date="2022-11-21"/>
+-     <release version="v0.5" date="2022-11-16"/>
++     <release version="v0.5" date="2022-11-16">
```
Comment (hfiguiere) [inline fix_metadata_release.patch:12]: this lead to invalid xml

File: io.github.kevinsmia1939.PySimpleCV.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      - install -Dm644 data/logo/logo.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dm644 data/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+    sources:
+      - type: archive
+        url: https://github.com/kevinsmia1939/PySimpleCV/archive/refs/tags/v0.5.3.tar.gz
+        sha256: 67408e0b9bc46c825b3c58a0b884fcf1ad58703c2c161a0cf45df8604bda686b
+        # Fix upstream
+      - type: patch
+        path: fix_metadata_release.patch
+        x-checker-data:
```
Comment (hfiguiere) [inline io.github.kevinsmia1939.PySimpleCV.yaml:99]: this is misplaced, you attached it to the patch instead of the main source.

**Comment by kevinsmia1939** [comment]
bot, build io.github.kevinsmia1939.PySimpleCV

**Comment by kevinsmia1939** [comment]
bot, build io.github.kevinsmia1939.PySimpleCV

**Comment by kevinsmia1939** [comment]
bot, build io.github.kevinsmia1939.PySimpleCV

**Comment by kevinsmia1939** [comment]
bot, build io.github.kevinsmia1939.PySimpleCV

**Comment by kevinsmia1939** [comment]
The app works fine.

**Comment by kevinsmia1939** [comment]
I think it is ready.

**Comment by barthalion** [comment]
bot, build io.github.kevinsmia1939.PySimpleCV

**Pull request closed without merging**